config/root/clusterworkspace-default.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: tenancy.kcp.dev/v1alpha1
-kind: ClusterWorkspace
-metadata:
-  name: default
-spec:
-  type:
-    name: Organization
-    path: root
```

Comment:
if this is deleted, is there an update to the admin.kubeconfig we produce in pkg/server/options/authentication? There should.

Reply:
I'll have to figure out a way to add this that's not blocking for starting our controllers.

Reply:
just delete it and fix the admin.kubeconfig to point to root

Reply:
Done